### PR TITLE
add map to userProfile with maxRange

### PR DESCRIPTION
**Pull request by anhuwi**
ATENTION: server change needed as well (get longitude and latitude from user)
add Map to userProfile

### CODE REVIEW

File: src/components/views/UserPage.js [hunk 1]
```diff
@@ -153,6 +158,35 @@ const UserPage = () =>{
         )
     }
 
+    // MAP
+    let map = null;
+    let position = [0, 0];  // default position
+    let zoomScale = 14;     // set zoom
+    if(user){
+        position = [user.latitude, user.longitude]     // latitude, longitude
+        if(position[0]===0){
```
Comment (serainaschraff) [inline src/components/views/UserPage.js:167]: shouldn't this be tested in both directions? Yes it is highly unlikely that somone manages to be exactly on the equator or exactly on the greenwich line but still..

**Review by serainaschraff — APPROVED** [review]
looks great :) even though it would make sense to have it on the edit preference page I would not put it there because of the card look that we have going on there..

File: src/components/views/UserPage.js [hunk 2]
```diff
@@ -153,6 +158,35 @@ const UserPage = () =>{
         )
     }
 
+    // MAP
+    let map = null;
+    let position = [0, 0];  // default position
+    let zoomScale = 14;     // set zoom
```
Comment (daveking7) [inline src/components/views/UserPage.js:164]: I think zoomScale should depend on the range!

File: src/styles/views/UserPage.scss [hunk 3]
```diff
@@ -1,6 +1,15 @@
 @import 'src/styles/theme';
 
 .userPage {
+  .leaflet-container {
+    width: 60%;
+    height: 60vh;
```
Comment (daveking7) [inline src/styles/views/UserPage.scss:6]: In my opinion the map should always be a square, no matter the zoom

Comment (daveking7) [reply src/styles/views/UserPage.scss:6]: Or maybe not, but your combination does not work

**Review by daveking7 — CHANGES_REQUESTED** [review]
Wait until my branch is pulled into this

**Comment by anhuwi** [comment]
> Wait until my branch is pulled into this

will do

**Review by daveking7 — APPROVED** [review]
Now it's fine

**Comment by anhuwi** [comment]
> Now it's fine

i thought about make a ui file "Map" where the map is defined, to increase readability of profilpage. but we only use it once.

**Pull request merged**